storage/src/main/java/org/apache/kafka/storage/internals/log/RemoteIndexCache.java

```diff
@@ -134,7 +134,7 @@ public RemoteIndexCache(int maxSize, RemoteStorageManager remoteStorageManager,
                 .maximumSize(maxSize)
                 // removeListener is invoked when either the entry is invalidated (means manual removal by the caller) or
```

Comment:
Can we update the comment to refer the `evictionListener` instead of `removeListener`?

Reply:
Done